acceptance/setup/pre_suite/90_install_devel_puppetdb.rb

```diff
@@ -41,6 +40,11 @@
   when :git
     install_puppetdb_termini_via_rake(master, databases)
   when :package
-    install_puppetdb_termini(master, databases)
+    os = PuppetDBExtensions.config[:os_families][master.name]
+    if test_config[:install_mode] == :upgrade_oldest && ([:redhat, :fedora].include? os)
+      install_puppetdb_termini(master, databases, nil, 'puppetdb-terminus')
+    else
```

Comment:
we don't want to bail under any circumstances here, since the possible install_modes are validated in helper.rb, but 'm going to hold off on any changes to this since I think https://github.com/puppetlabs/puppetlabs-puppetdb/pull/220 is a better way to fix our issue once I fix some ruby 1.8.7 issue I just noticed.